artichoke-backend/src/extn/core/kernel/require.rs

```diff
@@ -281,8 +281,8 @@ impl RelativePath {
     }
 }
 
-fn load_error(interp: &Artichoke, filename: &[u8]) -> Result<LoadError, Exception> {
-    let mut message = String::from("cannot load such file -- ");
-    string::format_unicode_debug_into(&mut message, filename)?;
-    Ok(LoadError::new(interp, message))
+fn load_error(interp: &Artichoke, filename: &[u8]) -> LoadError {
+    let mut message = String::from("cannot load such file -- ").into_bytes();
```

Comment:
As a matter of style, I dislike creating a `String` and "unparsing" into a `Vec` when we can create the `Vec` directly.

Reply:
```suggestion
    let mut message = b"cannot load such file -- ".to_vec();
```